components/authz-service/server/v2/authz.go

```diff
@@ -139,6 +140,10 @@ func (s *authzServer) FilterAuthorizedPairs(
 func (s *authzServer) FilterAuthorizedProjects(
 	ctx context.Context,
 	req *api.FilterAuthorizedPairsReq) (*api.FilterAuthorizedProjectsResp, error) {
+
+	// Introspection needs unfiltered access.
+	ctx = auth_context.ContextWithoutProjects(ctx)
```

Comment:
Consider moving this to just after 155 since that's where we use it (I got confused and thought `V2FilterAuthorizedProjects`

Reply:
🤔 Do we need this in `FilterAuthorizedProjects` at all?  I wonder if we really want to push it into `getAllowedProjects` (which calls `FilterAuthorizedProjects`) and is what `IntrospectAllProjects` calls.

Reply:
(or `IntrospectAllProjects` itself)

Reply:
I guess it's all the same flow so 🤷‍♂ nvmd